Dockerfile

```diff
@@ -1,8 +1,7 @@
 FROM golang:1.11
 
-ENV PATH /go/bin:/usr/local/go/bin:$PATH
-ENV GOPATH /go
-
 RUN apt-get update && apt-get install -y \
     libsystemd-dev \
     init
+
+RUN go get -u golang.org/x/lint/golint
```

Comment:
Didn't you just remove this in the last commit? We should clean up the commit history here a bit so that it is more easily reviewable.